src/extension/devtools/devtools.ts

```diff
@@ -122,29 +225,20 @@ devtools.listen<string>(CREATE_DEVTOOLS_PANEL, async ({ payload }) => {
       `background:tab-${inspectedTabId}:client`
     );
 
-    // Listen for tab reload from background
-    removeReloadListener = devtools.listen(RELOADING_TAB, () => {
```

Comment:
The notion of "tab reloading" has gone away in favor of thinking about this as a "disconnect" event. The user may navigate away from the current application rather than reloading the page which would fire the same event.